modules/programs/keychain.nix

```
@@ -84,6 +84,14 @@ in
         Whether to enable Zsh integration.
       '';
     };
+
+    enableXsessionIntegration = mkOption {
+      default = true;
+      type = types.bool;
+      description = ''
+        Whether to run keychain from your ~/.profile.
```

Shouldn't it be `~/.xsession`? Or rather,
```suggestion
        Whether to run keychain from your <filename>~/.xsession</filename>.
```

---

You're right, yeah